tests/unit/backend/corpora/common/entities/test_collection.py

```diff
@@ -197,3 +197,23 @@ def test__reshape_for_api(self):
         test_collection = Collection.create(self.session, **BogusCollectionParams.get())
         response = test_collection.reshape_for_api()
         self.assertEqual([], response["datasets"])
+
+    def test_tombstone_collection_tombstones_all_datasets_in_collection(self):
+        collection = self.generate_collection(
+            self.session, visibility=CollectionVisibility.PRIVATE.name, owner="test_user_id"
+        )
+        self.generate_dataset(self.session, collection=collection)
+        self.generate_dataset(self.session, collection=collection)
+
+        self.assertEqual(len(collection.datasets), 2)
```

Comment:
Remove. We don't need to test this again.

Reply:
see above comment about testing existence before testing for deletion

Reply:
but you can and are retrieving the rows and can see that they are tombstoned after the delete. This confirms that they were both created and tombstoned. I still think this extra verify is superfluous. What you could add `tombstone=False` to the dataset and collections to make sure you didn't generate a tombstoned dataset or collection from the start. Then you could remove these extra lines from the tests. I guess this is a bit nit. I guess it's fine either way.